### PR TITLE
feat(fxa-56): add fxa-56 doc

### PR DESCRIPTION
**Pull request by vladikoff**

### CODE REVIEW

File: features/FxA-56-email-deliverability/README.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+## Outcomes
+
+To have better metrics and logging around our email delivery, with
+the goal of improving email deliverability afterwards.
+
+## Hypothesis
+
+If we add metrics to email services in FxA then
+**we will have a chance to improve the email deliverability** of the
+authenticate system.
```
Comment (rfk) [inline features/FxA-56-email-deliverability/README.md]: nit: authentication

File: features/FxA-56-email-deliverability/README.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+If we stop sending additional email messages to bounced emails addresses then our "complaint rate" with popular email providers will decrease and improve our deliverability.
+
+
+## Metrics
+
+The following metrics should help us understand and debug email deliverability:
+
+* Flow events for email bounces
+* Current bounce rate
+* A dashboard that lists common
```
Comment (rfk) [inline features/FxA-56-email-deliverability/README.md]: incomplete sentence here "lists common..."

**Comment by rfk** [comment]
@vladikoff @seanmonstar let's sync up early next week to push this forward.

**Comment by vladikoff** [comment]
from mtg: seems like we have most things documented here, @rfk to do last review and merge...

**Pull request merged**